src/js/utils/ui.js

```diff
@@ -112,6 +112,7 @@ function initInteractionListeners(ui) {
             if (!passive) {
                 const { pointerId } = e;
                 ui.pointerId = pointerId;
+                el.setPointerCapture(pointerId);
```

Comment:
no safari support, i assume this is a known?

also really cool api! haven't seen this before

Reply:
ui.js normalizes input across browsers for Mouse, Touch and Pointer events. It listens for Mouse or Pointer events and Touch events when available.